google/cloud/pubsub/topic_admin_client.h

```diff
@@ -228,9 +225,6 @@ class TopicAdminClient {
    * @par Idempotency
    * This is a read-only operation and therefore always idempotent and retried.
    *
-   * @par Example
-   * @snippet samples.cc list-topic-snapshots
```

Comment:
same

Reply:
Because this is the handwritten API. It should just be deprecated.

Reply:
That is reasonable. The sample with the generated client might show customers how to update their code. I don't care either way. But whatever you decide to do should be consistent:

https://github.com/googleapis/google-cloud-cpp/blob/4ab32bf4d4eb6c9b0870ff45493d676352a43a49/google/cloud/pubsub/topic_admin_client.h#L108

^ here we reference the sample with the generated client.

Reply:
Yeah that's fair. 

removed.